DARPA-ASKEM/terarium
# update default config also updates model

### Summary
This PR does two things
- Allows default configuration to be changed
- When changing default config, the change is propagated into the model. This introduces a refreshing bug where model's description page is not refreshed properly. I think rather than trying to force a workaround here, we should just wait for the model page refactor/restructure that we plan to solve this, and other issues.

### Testing
Open up a model
- Go to the model tab
- Change a parameter's value for Default config
- Refresh the page, note that a) The change is persisting, and b) the Description page reflects the same change
